editions/tw5.com/tiddlers/Deprecated core macros.tid

```diff
@@ -0,0 +1,26 @@
+created: 20251101085817414
+modified: 20251101091035398
+tags: [[Core Macros]] Macros $:/deprecated
+title: Deprecated core macros
+type: text/vnd.tiddlywiki
+
+<<.warning "It is discouraged to use the following macros. Though tiddlywiki might still support them, they are in the process of being dropped, or may only be kept for compatibility purposes. Avoid using it, and update existing code if possible. Be aware that these macros may cease to work at any time. ">>
+
+Most deprecated macros are defined in [[$:/core/macros/deprecated]]. It is discouraged to use them.
+
+! Stylesheet Macros
+
+<<.deprecated-since 5.4.0>> The following core [[macros|Macros]] used to made it easy to specify alternative browser-specific properties when constructing a [[stylesheet|Cascading Style Sheets]] tiddler. They are deprecated after 2017 baseline is supported in v5.4.0:
+
+;`<<box-shadow shadow>>`
+: for the `x-box-shadow` properties
```

Comment:
Please add **linebreaks** between the different DL elements **and** a space in front of `;`
So ``; `<<box-shadow `` ... This makes the source plain text more human readable

eg:

```
; term
: definition

; term 
: definition
```